pages/chaoss/visualizations/contrib_importance_pie.py

```diff
@@ -70,6 +71,7 @@
                                             options=[
                                                 {"label": "Commit", "value": "Commit"},
                                                 {"label": "Issue Opened", "value": "Issue Opened"},
+                                                {"label": "Issue Closed", "value": "Issue Closed"},
                                                 {"label": "PR Open", "value": "PR Open"},
```

Comment:
Add option for "Issue Comment" or group together pr and issue comment as "Comment". example of this can be seen in contributors/contrib_by_action.py. Line of code that makes it work: 

`df = df[df["Action"].str.contains(action)]`